jenkins/shared-library/vars/githubHelper.groovy

```diff
@@ -68,15 +55,11 @@ Object createPullRequest(
 }
 
 Object requestReviewersForPullRequest(String token, String ownerName, String repositoryName, int pullRequestNumber, List reviewers) {
-	final String reviewersCommand = """
-		curl -L \\
-			-X POST \\
-			-H "Accept: application/vnd.github+json" \\
-			-H "Authorization: Bearer ${token}" \\
-			-H "X-GitHub-Api-Version: 2022-11-28" \\
-			https://api.github.com/repos/${ownerName}/${repositoryName}/pulls/${pullRequestNumber}/requested_reviewers \\
-			-d '{"reviewers": ["${reviewers.join('", "')}"]}'
-	"""
+	final String reviewersCommand = new StringBuilder().append('curl -L -X POST -H "Accept: application/vnd.github+json" ')
+		.append("""-H "Authorization: Bearer ${token}" """)
+		.append('-H "X-GitHub-Api-Version: 2022-11-28" ')
```

Comment:
i'd also consider refactoring this since these headers seem to be same for all github requests (only variable is method GET/POST, which can be argument)